documentation/modules/auxiliary/dos/http/wordpress_directory_traversal_dos.md

```diff
@@ -0,0 +1,31 @@
+This module exploits a Cross-site request forgery (CSRF) vulnerability in the wp_ajax_update_plugin function in wp-admin/includes/ajax-actions.php in Wordpress before 4.6. Allows remote authenticated users to cause a denial of service (with /dev/random read operations).
+## Verification
+
+1. Start msfconsole
+2. Do: ```use auxiliary/dos/http/wordpress_directory_traversal_dos.rb```
+3. Do: ```set RHOST <ip target site>```
+4. Do: ```set TARGETURI <WordPress path>```
+5. Do: ```set USERNAME <Valid Username>```
+6. Do: ```set PASSWORD <Valid Password>```
+7. Do: ```exploit```
+8. WordPress website should be down
+
+## Scenarios
```

Comment:
Add a 3rd level heading with details of what was tested here which can help for debugging it in 5yrs or so if someone had to come back and look.
```
### Wordpress 4.5.9 on Ubuntu 16.04.02
```